README.md

```diff
@@ -66,34 +66,40 @@ Install the ``python3-routes`` package:
 
 These instructions are meant for developers who are compiling the code for
 development and testing. To build binaries that are suitable for installation
-we recommend that you build .deb or .rpm packages, or refer to ``ceph.spec.in``
-or ``debian/rules`` to see which configuration options are specified for
-production builds.
+we recommend that you build `.deb` or `.rpm` packages, or refer to
```

Comment:
s/or/and/ ?

Reply:
I think that this is referring to either the Debian-related or the Fedora-related packaging systems, but I suppose that you could think of this as as example of "inclusive or". I'm going to let it stand as it is.